src/data/markdown/translated-guides/en/02 Using k6/02 Metrics.md

```diff
@@ -69,16 +69,80 @@ All the `http_req_...` lines and the ones after them are _built-in_ metrics that
+| iteration_duration | Trend   | The time it took to complete one full iteration. It includes the time spent in `setup` and `teardown` as well, there is currently a workaround for excluding them shown in the example below. |
 | dropped_iterations | Counter | Introduced in k6 v0.27.0, the number of iterations that could not be started due to lack of VUs (for the arrival-rate executors) or lack of time (due to expired maxDuration in the iteration-based executors). |
-| data_received      | Counter | The amount of received data. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL.                                                                                 |
-| data_sent          | Counter | The amount of data sent. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL.                                                                                     |
-| checks             | Rate    | The rate of successful checks.                                                                                                                                                                                  |
+| data_received      | Counter | The amount of received data. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL. |
+| data_sent          | Counter | The amount of data sent. Read [this example](/examples/track-transmitted-data-per-url) to track data for an individual URL. |
+| checks             | Rate    | The rate of successful checks. |
+
+<Collapsible title="Workaround to isolate `iteration_duration` metric from Setup and Teardown">
+
+A common requested case is to track the `iteration_duration` metric without including time spent for `setup` and `teardown` functions.
+This feature is not yet available but a threshold on `iteration_duration` can be used as a workaround. Check the following example.
+
+<CodeGroup lineNumbers={[true]}>
+
+```javascript
+import { sleep } from 'k6'
+import http from 'k6/http'
+
+export const options = {
+    vus: 1,
+    iterations: 1,
+    thresholds: {
+        // We're using 'scenario:default' because that's the internal k6
+        // scenario name when we haven't actually specified `options.scenarios`
+        // explicitly and are just using the old execution shortcuts instead...
+        'iteration_duration{scenario:default}': [
+            // Some dummy threshold that's always going to pass.
+	    // You don't even need to have something here.
+            `max>=0`,
+        ],
+        // You can isolate setup and teardown metrics, if you are interested in them
+        // for some reason. This looks nasty, but '::' is the group separator.
+        'iteration_duration{group:::setup}': [`max>=0`],
+        'iteration_duration{group:::teardown}': [`max>=0`],
+
+        // And you can also isolate the HTTP metrics, in case of long-running requests
```

Comment:
```suggestion
        // And you can also remove the time spent on the HTTP metrics, in case of long-running requests
```
Correct?

I think "isolated" is misleading as this workaround doesn't actually isolate anything: it just allows one to get additional values of duration with specific conditions.

Reply:
I kind of feel `isolate` is better, as it doesn't remove any metrics it just looks it just aggregates only metrics that match. 

Maybe : 
```suggestion
        // And you can do this for all metrics, for example including HTTP metrics
```
and then have a more concrete explanation above?